docs/sources/configure-client/profile-types.md

```diff
@@ -0,0 +1,107 @@
+| Goroutines     | Yes       |      |           |               |
+| Mutex Count    |           |      |           |               |
+| Mutex Duration |           |      |           |               |
+| Block Count    | Yes       |      |           |               |
+| Block Duration | Yes       |      |           |               |
+| Lock Count     |           | Yes  |           |               |
+| Lock Duration  |           | Yes  |           |               |
+| Exceptions     |           |      |           |               |
+| Wall           |           |      |           |               |
+| Heap           |           |      |           |               |
+
+### Instrumentation with SDKs
+
+Using the Pyroscope language SDKs lets you instrument your application directly for precise profiling. You can customize the profiling process according to your application’s specific requirements.
+
+For more information on the language SDKs, refer to [Pyroscope language SDKs](https://grafana.com/docs/pyroscope/<PYROSCOPE_VERSION>/configure-client/language-sdks/).
+
+This table lists the available profile types based on the language SDK.
+
+| Profile type   | Go (push) | Java | .NET       | Ruby | Python | Rust | Node.js |
+| -------------- | --------- | ---- | ---------- | ---- | ------ | ---- | ------- |
+| CPU            | Yes       | Yes  | Yes        | Yes  | Yes    | Yes  | Yes     |
+| Alloc Objects  | Yes       | Yes  | Yes        |      |        |      |         |
+| Alloc Space    | Yes       | Yes  | Yes        |      |        |      |         |
+| Inuse Objects  | Yes       |      | Yes (7.0+) |      |        |      |         |
+| Inuse Space    | Yes       |      | Yes (7.0+) |      |        |      |         |
+| Goroutines     | Yes       |      |            |      |        |      |         |
+| Mutex Count    | Yes       |      | Yes        |      |        |      |         |
+| Mutex Duration | Yes       |      | Yes        |      |        |      |         |
+| Block Count    | Yes       |      |            |      |        |      |         |
+| Block Duration | Yes       |      |            |      |        |      |         |
+| Lock Count     |           | Yes  | Yes        |      |        |      |         |
+| Lock Duration  |           | Yes  | Yes        |      |        |      |         |
+| Exceptions     |           |      | Yes        |      |        |      |         |
+| Wall           |           |      | Yes        |      |        |      |         |
+| Heap           |           |      | Yes (7.0+) |      |        |      | Yes     |
+
+## Profile types supported with span profiles
+
+Pyroscope can integrate with distributed tracing systems supporting the OpenTelemetry standard. This integration lets you link traces with the profiling data and find resource usage for specific lines of code for your trace spans.
+
+This table lists the available profile types for span profiles.
```

Comment:
Yes. That was a left over from when I had a table and realized only one profile type was supported, so easier to list things instead of having a table.